sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -81,97 +82,86 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 To create a pipeline topology you need to define parameters, sources, and sinks.
 
-```C# Snippet:Azure_VideoAnalyzerces_Samples_SetParameters
-// Add parameters to Topology
-private void SetParameters(PipelineTopologyProperties pipelineTopologyProperties)
+### Define Parameters
+
+```C# Snippet:Azure_VideoAnalyzerSamples_SetParameters
+pipelineTopologyProperties.Parameters.Add(new ParameterDeclaration("rtspUserName", ParameterType.String)
+{
+    Description = "rtsp source user name.",
+    Default = "dummyUserName"
```

Comment:
You should not use "dummy" in public docs or code. This was/is actually a Microsoft writing style guideline. "fake" or "test" or something else would be better.

Reply:
Ahh sorry, thought I had changed all of the instances of 'dummy'. Thanks for finding this. Made sure to do a full search this time :)